docs/MarkdownInput.md

```diff
@@ -30,4 +29,165 @@ const PostEdit = () => (
 );
 ```
 
+## Props
+
+`<MarkdownInput>` accepts the following props:
+
+| Prop                 | Required | Type      | Default    | Description                                                           |
+| -------------------- | -------- | --------- | ---------- | --------------------------------------------------------------------- |
+| `source`             | Required | `string`  |            | The field name in the record                                          |
```

Comment:
Props should be in alphabetic order in this table, stating with required props.
Ditto for the rest